src/NuGet.Core/NuGet.Packaging.Extraction/Strings.resx

```diff
@@ -341,7 +341,7 @@ Valid from:</comment>
     <value>The package hash uses an unsupported hash algorithm.</value>
   </data>
   <data name="SignaturePackageIntegrityFailure" xml:space="preserve">
-    <value>The package integrity check failed.</value>
+    <value>The package integrity check failed. The package has changed since it was signed. Try clearing the local cache and run nuget operation again.</value>
```

Comment:
I think the term local cache is ambiguous. 

Are we referring to the http cache? global packages folder? 

Note the specific language in https://docs.microsoft.com/en-us/nuget/consume-packages/managing-the-global-packages-and-cache-folders

Reply:
Great question Nikolche. I have not thought about that. I produced this string based on content in https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3008#solution. I will take a step back and learn more about it.

Reply:
Reading from the [package-installation-process doc](https://docs.microsoft.com/en-us/nuget/concepts/package-installation-process), packages are searched in the following order during install.

 `global packages folder` -> `fallback folders` -> `local sources` -> `http-cache` ->`online sources`.

My understanding is when a package is fetched from an online source it will be cached in the `http-cache` first and then perform `package signature validation` before saving a copy in the `global packages folder`. If the package fails signature validation because the contents have been changed since it was signed, then the package will not be saved to the global packages folder but remain in the `http-cache`. When we try to install again the same package before http-cache gets invalidated, then the package will be fetched from the `http-cache`. Hence user may get the same warning again. 

Given the above details, I would like to change the warning message to `The package integrity checks failed. The package has changed since it was signed. Try clearing the local http-cache and run nuget operation again.`

Reply:
Right on all accounts :) 
Great job, I agree with your proposal.